frontend/src/js/controllers/authCtrl.js

```diff
@@ -244,10 +244,9 @@
                 parameters.callback = {
                     onSuccess: function(response) {
                         var status = response.status;
```

Comment:
remove this line since it's not used anywhere within it's scope. (`var status = response.status;`)